content/docs/1.5.0/spdk/prerequisites.md

```diff
@@ -5,11 +5,19 @@ weight: 2
 
 ## Prerequisites
 
+Longhorn worker nodes must meet the following requirements:
```

Comment:
What does Longhorn worker node mean? Do you know nodes running with instance manager?

Reply:
Yes.